### PR TITLE
Working branch

### PR DESCRIPTION
**Pull request by Grifo89**
Hey guys, this pull request is for code review, The website seems stable and right could you check it out? Thanks.

### CODE REVIEW

**Review by Ispirett — CHANGES_REQUESTED** [review]
Good work so far @Grifo89  :+1: :100: 

This is how your project looks to me so far.
please address this issue.
it should look more like the original site.

current

**Reminder use  more descriptive messages in pr description 
Reminder use descriptive  commit messages.**

**Comment by Grifo89** [comment]
Hey, Guys, I don know why when you click on the index file the images don't show up, but when you run the project with a server the page is ok. But the way I think that is because I have the images into a folder I don't know.
image

File: style.css [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+  overflow: auto;
+}
+
+#nav-support {
+  background: url(images/globalnav_text.png) no-repeat 50% 101.2%;
+  overflow: auto;
+}
+
+.hero-container {
+  background: url(/images/your_verse_hero.jpg) no-repeat 50% 100%;
```
Comment (bolah2009) [inline style.css]: Let's remove the first backslash `/`.

```suggestion
  background: url(images/your_verse_hero.jpg) no-repeat 50% 100%;
```

> To reference a file in a subdirectory, write the directory name in front of the path, plus a forward slash, e.g. subdirectory/my-image.jpg.

Learn more [here](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/Dealing_with_files#File_paths)

### Please do this for all other occurrences.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+  overflow: hidden;
+}
+
+.img {
+  display: grid;
+  grid-template: 1fr/ 1fr 1fr 1fr 1fr;
+}
+
+.img1 {
+  background: url(/images/promo_iphone5s.jpg);
```
Comment (bolah2009) [inline style.css]: Let's remove the first backslash `/`.

```suggestion
  background: url(images/promo_iphone5s.jpg);
```

File: style.css [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+  margin: 0 1px;
+}
+
+.img2 {
+  background: url(/images/promo_iphone5c.jpg);
```
Comment (bolah2009) [inline style.css]: Let's remove the first backslash `/`.

```suggestion
  background: url(images/promo_iphone5c.jpg);
```

File: style.css [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+  margin: 0 1px;
+}
+
+.img3 {
+  background: url(/images/promo_30_years.jpg);
```
Comment (bolah2009) [inline style.css]: Let's remove the first backslash `/`.

```suggestion
  background: url(images/promo_30_years.jpg);
```

File: style.css [hunk 5]
```diff
@@ -0,0 +1,326 @@
...
+  margin: 0 1px;
+}
+
+.img4 {
+  background: url(/images/promo_supplier_responsibility.jpg);
```
Comment (bolah2009) [inline style.css]: Let's remove the first backslash `/`.

```suggestion
  background: url(images/promo_supplier_responsibility.jpg);
```

File: index.html [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+        </ul>
+        <form class="nav-form">
+          <i class="fas fa-search"></i>
+          <input class="input"
+          type="text" name="search" value="" placeholder="Search">
+        </form>
+      </nav>
+      <div class="hero-container">
+        <div class="hero">
+          <h1 class="title"><a href="#"><img src="images/your_verse_title_small.png" alt=""></a></h1>
```
Comment (bolah2009) [inline index.html]: Consider adding a text and hide the visibility with CSS. :thinking: 

#### [See HTML Validation warning](https://validator.w3.org/nu/?doc=https%3A%2F%2Frawcdn.githack.com%2FGrifo89%2FApple_Clone%2F3f384fbd8b277592255da8e5dca6365aa8a0975f%2Findex.html).

File: index.html [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+          <h2 class="subheader">
+            <a href="#"><img src="images/your_verse_subhead_small.png" alt=""></a>
+          </h2>
```
Comment (bolah2009) [inline index.html]: Consider adding a text and hide the visibility with CSS. :thinking:

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Awesome job @Grifo89 

#### [The clone](https://rawcdn.githack.com/Grifo89/Apple_Clone/3f384fbd8b277592255da8e5dca6365aa8a0975f/index.html#) looks good to me, and uses CSS Backgrounds and gradient effectively. 

However, please do the following before a :+1: 
- Fix broken image links. See comments
- Fix[ HTML Validation warnings](https://validator.w3.org/nu/?doc=https%3A%2F%2Frawcdn.githack.com%2FGrifo89%2FApple_Clone%2F3f384fbd8b277592255da8e5dca6365aa8a0975f%2Findex.html)
- [OPTIONAL] Have you tried using CSS for the effects before considering Javascript? :thinking: 

### Let's make this Pull Request look more professional.
  - [NEXT PROJECT]: Use meaningful commits messages. [Learn more here](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) and Check [this awesome article](https://chris.beams.io/posts/git-commit/)
  - [REQUIRED] Use meaningful names for  Pull Requests.
  - [REQUIRED] Add a short summary of changes in the Pull Request description.
- [NEXT PROJECTS]: For the next (HTML/CSS) project, please follow `GitHub flow` [guidelines](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/). What is missing here is a descriptive branch name instead of `workingBranch`.


#### :recycle:  Make these changes and submit another code review request. :+1: 

#### Feel free to reach me on slack if you having issues implementing these features. :+1: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Comment by Grifo89** [comment]
Hey @bolah2009 thank you so much for the awesome feedback you've given us so far. I already made what you suggested to me. You have been the best reviewer we've had so far as well.

**Review by Forison — COMMENTED** [review]
# Pull request under review by Addo

File: style.css [hunk 8]
```diff
@@ -0,0 +1,326 @@
...
+  background: url(images/globalnav_text.png) no-repeat 50% 86%;
+  overflow: auto;
+}
+
+#nav-support {
+  background: url(images/globalnav_text.png) no-repeat 50% 101.2%;
+  overflow: auto;
+}
+
+.hero-container {
```
Comment (Forison) [inline style.css:245]: image displays when i change path to this 
background: url(./images/globalnav_text.png) no-repeat 50% 101.2%;

File: style.css (hunk 3 above)
Comment (Forison) [inline style.css]: image displays when i change path to this 
background: url(./images/globalnav_text.png) no-repeat 50% 101.2%;

File: style.css (hunk 4 above)
Comment (Forison) [inline style.css]: ## image displays when i change path to this 
background: url(./images/globalnav_text.png) no-repeat 50% 101.2%;

File: style.css (hunk 5 above)
Comment (Forison) [inline style.css]: ## image displays when i change path to this 
background: url(./images/globalnav_text.png) no-repeat 50% 101.2%;

File: style.css [hunk 9]
```diff
@@ -0,0 +1,326 @@
...
+  background: url(/images/promo_iphone5s.jpg);
+  margin: 0 1px;
+}
+
+.img2 {
+  background: url(/images/promo_iphone5c.jpg);
+  margin: 0 1px;
+}
+
+.img3 {
```
Comment (Forison) [inline style.css:284]: ## image displays when i change path to this 
background: url(./images/globalnav_text.png) no-repeat 50% 101.2%;

**Review by Forison — CHANGES_REQUESTED** [review]
## Hi @Grifo89 🙂🙂
## This is what I  get when I try to view your page😔 
Screenshot from 2019-10-01 17-15-16
As suggested by the previous TSE you need to correct the paths that you gave your background URL 

- Again the HTML validations warnings still persist (From line 33, column 11; to line 33, column 28) and (From line 34, column 11; to line 34, column 32). visit [ validator ](https://validator.w3.org)when uploading the index file.🙂

- Background gradient implementation on navbar not apparent
> I realized that you were using background images on the navbar links instead of applying the background gradient. please refer to this [article](https://www.w3schools.com/css/css3_gradients.asp)
> apply the background gradient on the parent navbar,  give each link a border-right, apply CSS hover on each link and voila...

> always remember to give the ```alt``` attribute of ```img``` a value. It is very important for SEO 

## Kindly apply the changes suggested and submit code for review
Happy coding🙂🙂

**Comment by Grifo89** [comment]
Sorry, I forgot to push my last commit. It's already done and but the way sure I'm using gradients for the navbar as a background the image is just for the words 😕

**Review by stratospherique — APPROVED** [review]
### Nice work implementing the suggested changes :clap: . Your project is approved :heavy_check_mark:  and you may merge to master :+1: :+1: :+1:

**Pull request merged**